### PR TITLE
Added .sql files to check SQL User permissions.

### PR DESCRIPTION
**Pull request by amoghrm**
* We can ask customer/user to run these scripts to fetch all tables and colums, all_relations for the db, and the schema list.

### CODE REVIEW

**Review by arpit-lb — APPROVED** [review]

File: sql_user_check/all_tables_columns.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
+SELECT
```
Comment (aditya-lb) [inline sql_user_check/all_tables_columns.sql:1]: This query can return a lot of results and we are not exactly using the result for downstream processes. Can we not do something like `SELECT 1 FROM ....` instead? Ditto below as well.

File: sql_user_check/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Please run each of the .sql files on the User and Database that you need to connect to LightBeam.
```
Comment (aditya-lb) [inline sql_user_check/README.md:1]:
```suggestion
Please run each of the .sql files on that database with the account credentials that you need to connect to LightBeam.
```

**Review by tushar-lb — APPROVED** [review]
LGTM. I think it would be good if we have one shell script which will take the required parameters as input and execute the SQL scripts one after another and generate an output file with the results of all the queries.

File: sql_user_check/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Refer the examples below.
+
+Fetch Schema List:
+psql -h <DATABASE_HOST_IP> -U <DATABASE_USER> -f schema_list.sql -p <PORT> -d <DATABASE_NAME> > schema_list
```
Comment (aditya-lb) [inline sql_user_check/README.md:5]: Use the sql markdown syntax so that README.md can display it as a sql query instead. Ditto below as well.

**Review by aditya-lb — DISMISSED** [review]

**Pull request merged**